src/resources/vault-locations/VaultLocationList.tsx

```diff
@@ -21,10 +21,6 @@ export default function VaultLocationList(): React.ReactElement {
     [record]
   )
 
```

Comment:
Hello @TahaKhanAbdalli  - I've tried calling the `setOpen` state entity directly from the TSX.

It simplifies the code, but is it ok to do that?